src/modules/item/export.ts

```diff
@@ -153,14 +154,14 @@ function buildItemEntityMetadata(collection: Collection, item: Item): StandardCa
     name: item.name,
     description: item.description,
     collectionAddress: collection.contractAddress!,
-    rarity: item.rarity!,
-    i18n: [{ code: 'en', text: item.name }],
+    rarity: (item.rarity! as unknown) as Rarity,
+    i18n: [{ code: 'en', text: item.name }] as I18N[],
```

Comment:
maybe we can save the casting here and use `Locale` everywhere?

```
i18n: [{ code: Locale.EN, text: dbItemMock.name }],
```

Reply:
Done!